samples/automl/automlVisionDataset.js

```diff
@@ -0,0 +1,397 @@
+        console.log(`\n`);
+      }
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_listDatasets]
+}
+
+function getDataset(projectId, computeRegion, datasetId) {
+  // [START automl_vision_getDataset]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset`;
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Get complete detail of the dataset.
+  client
+    .getDataset({name: datasetFullId})
+    .then(responses => {
+      const dataset = responses[0];
+
+      // Display the dataset information.
+      console.log(`Dataset name: `, dataset.name);
+      console.log(`Dataset Id: `, dataset.name.split(`/`).pop(-1));
+      console.log(`Dataset display name: `, dataset.displayName);
+      console.log(`Dataset example count: `, dataset.exampleCount);
+      console.log(
+        `Classification type: `,
+        dataset.imageClassificationDatasetMetadata.classificationType
+      );
+      console.log(`Dataset create time: `);
+      console.log(`\tseconds: `, dataset.createTime.seconds);
+      console.log(`\tnanos: `, dataset.createTime.nanos);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_getDataset]
+}
+
+function importData(projectId, computeRegion, datasetId, path) {
+  // [START automl_vision_importData]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset`;
+  // const path = `string or array of .csv paths in AutoML Vision CSV format, e.g. “gs://myproject/traindata.csv”;`
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Get one or more Google Cloud Storage URI(s).
+  const inputUris = path.split(`,`);
+  const inputConfig = {
+    gcsSource: {
+      inputUris: inputUris,
+    },
+  };
+
+  // Import the dataset from the input URI.
+  client
+    .importData({name: datasetFullId, inputConfig: inputConfig})
+    .then(responses => {
+      const operation = responses[0];
+      console.log(`Processing import...`);
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log(`Data imported.`);

```

Comment:
Use brackets, and (if possible) drop the `=== true`:
```javascript
if (responses[2].done) {
  console.log(`Data imported.`);
}
```

Apply this comment throughout your PR.